app/managers/session_manager.ts

```diff
@@ -41,12 +42,15 @@ class SessionManager {
         if (Platform.OS === 'android') {
             AppState.addEventListener('blur', () => {
                 this.onAppStateChange('inactive');
+                callsOnAppStateChange('inactive');
```

Comment:
is it intended to have this on Android when the user is in the app switcher? or we just want it when the app switches app state as in foreground/background ?

Reply:
This is for anytime the rest of the app changes its state, then we want to cancel the ringing. It will only start ringing (when coming from background) when the new notification is mounted (so only when react starts updating again).